bedrock/settings/base.py

```diff
@@ -282,6 +282,7 @@ def _put_default_lang_first(langs, default_lang=LANGUAGE_CODE):
     "mozorg/about/shared",
     "navigation",
     "navigation_v2",
+    "m24-pencil-banner",
```

Comment:
**Nit:** suggest adding this to `/banners/` along with other banner string files?